CMakeLists.txt

```diff
@@ -26,6 +26,16 @@ include(GNUInstallDirs)
 include(FortranCInterface)
 include(CMakePackageConfigHelpers)
 
+# Requires OpenMP
```

Comment:
Right now erd isn't even taking in `ENABLE_OPENMP`. Does your comment mean "when OMP is enabled, add `-recursive`" or "only safe to link a program built with OMP=ON (i.e., Psi4) when ERD compiled with OMP enabled and `-recursive`"?